jobsdb/jobsdb.go

```diff
@@ -424,18 +422,6 @@ func (jd *HandleT) Setup(clearAll bool, tablePrefix string, retentionPeriod time
 		jd.addNewDS(appendToDsList, dataSetT{})
 	}
 
-	// Schema Migration: Created_at column should have a default now()
-	dList := jd.getDSList(false)
-	jd.setDefaultNowColumns(dList[len(dList)-1].Index)
```

Comment:
Shall we delete this function altogether?

Reply:
Yes, I will remove them.